Dockerfile

```diff
@@ -1,19 +1,25 @@
-FROM debian:buster-slim
+FROM debian:stretch
 LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends -q \
-    clang \
-    cmake \
-    git \
-    libssl-dev && \
-    make \
+                    clang \
+                    cmake \
+                    git \
+                    libssl-dev \
+                    make && \
     rm -rf /var/lib/apt/lists/*
 
 ADD . /libsxg
 RUN mkdir libsxg/docker_build -p && \
     cd libsxg/docker_build && \
-    cmake .. && \
-    make sxg
+    cmake .. \
+    	  -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Should these lines have one more space to align with the first argument, or should you use a fixed-width indent (say, of four columns)?

Reply:
Sorry, I used tab instead of space. Then I untabified all.